# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/70_mm-2.7-sendmail.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/bin/showexternal mm2.7/src/bin/showexternal
--- mm2.7.orig/src/bin/showexternal	2003-05-14 23:58:27.000000000 +0200
+++ mm2.7/src/bin/showexternal	2003-05-14 23:59:21.000000000 +0200
@@ -219,7 +219,7 @@
 	echo "" >> "$NEWNAME"
 	sed -e 1,/^\$/d < $bodyfile >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
-	/usr/lib/sendmail -t  < "$NEWNAME"
+	/usr/sbin/sendmail -t  < "$NEWNAME"
 	if ($status) then
 	    echo sendmail failed
 	    cd ${METAMAIL_TMPDIR}
